### PR TITLE
feature RTMD-90 - contact-details step

### PR DESCRIPTION
**Pull request by joefitter**
* added contact details step with tests

### CODE REVIEW

File: apps/rotm/translations/src/en/validation.json [hunk 1]
```diff
@@ -8,7 +8,20 @@
   "contact-consent": {
     "required": "Let us know whether we can contact you for more information"
   },
-  "contact-info-email": {
+  "name": {
+    "required": "Tell us your full name"
+  },
+  "contact-type": "Tell us how you would like to be contacted",
```
Comment (bendiggle) [inline apps/rotm/translations/src/en/validation.json]: Does this not to be in an object with the validation type?

File: apps/rotm/acceptance/features/contact-details.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  I,
+  contactDetailsPage,
+  confirmPage
+) => {
+  I.fillField(contactDetailsPage.id.name, contactDetailsPage.content.name);
+  I.checkOption(contactDetailsPage.id.emailOption);
+  I.fillField(contactDetailsPage.id.email, contactDetailsPage.content.validEmail);
+  I.submitForm();
+  I.seeInCurrentUrl(confirmPage.url);
+});
```
Comment (bendiggle) [inline apps/rotm/acceptance/features/contact-details.js]: lots of repeating. Could put these into functions in the helper.

File: apps/rotm/translations/src/en/validation.json (hunk 1 above)
Comment (joefitter) [inline apps/rotm/translations/src/en/validation.json]: yes, good spot!

File: apps/rotm/acceptance/features/contact-details.js (hunk 2 above)
Comment (joefitter) [inline apps/rotm/acceptance/features/contact-details.js]: yeah I know you do that, but I think I prefer the explicit nature of reading the repro steps in one file and just using pageObject for content. It's a preference thing though - @easternbloc @JoeChapman what say you?

Comment (joefitter) [inline apps/rotm/acceptance/features/contact-details.js]: If we get too programmatic it gets quite far from traditional acceptance testing

**Pull request merged**